src/runtime/driver/handle.rs

```diff
@@ -52,99 +53,29 @@ impl Handle {
         &self,
         buffers: Rc<RefCell<dyn FixedBuffers>>,
     ) -> io::Result<()> {
-        let mut driver = self.inner.borrow_mut();
-
-        driver
-            .uring
-            .submitter()
-            .register_buffers(buffers.borrow().iovecs())?;
-
-        driver.fixed_buffers = Some(buffers);
-        Ok(())
+        self.inner.borrow_mut().register_buffers(buffers)
```

Comment:
Lots of layers for getting register_buffers called now. Is this one really necessary or is this a stop gap version?

Reply:
I'm trying to keep everything which touches the driver's state contained to the driver.